packages/api/src/external/hie/command/get-hie-directory-entries.ts

```diff
@@ -12,17 +12,15 @@ export async function getHieDirectoryEntriesByFilter({
 }: {
   filter: string | undefined;
   pagination: Pagination;
-}): Promise<CQDirectoryEntry2[]> {
+}): Promise<HieDirectoryEntry[]> {
   const sequelize = HIEDirectoryEntryViewModel.sequelize;
   if (!sequelize) throw new Error("Sequelize not found");
 
   const querySelect = `SELECT * FROM ${HIEDirectoryEntryViewModel.NAME} WHERE 1 = 1 `;
 
   const queryFTS =
     querySelect +
-    (filter
-      ? ` AND (search_criteria @@ websearch_to_tsquery('english', :filter) OR id = :filter)`
-      : "");
+    (filter ? ` AND (search_criteria @@ websearch_to_tsquery('english', :filter))` : "");
```

Comment:
OOC, why removing the `or id = :filter` section of it?

Also, maybe worth having this code as a function so both the regular and the "count" queries can use the same logic?

Reply:
Yeah we could keep it in. Because the id is the same as the oid and we're including the oid in the FTS i dropped it from here. possible we should just do exact match on oid and thus drop it from the FTS instead.